README.md

```diff
@@ -32,6 +32,8 @@
 ### Key Features <a name="key-features"></a>
 
 - **[Create multiple classes]**
+- **[Add integration tests]**
+- **[Add API end points]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍